<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPrabhupadaLoginDialog</class>
 <widget class="QDialog" name="QPrabhupadaLoginDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>296</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Вход в &quot;Словарь Шрилы Прабхупады&quot;</string>
  </property>
  <property name="windowIcon">
   <iconset resource="PrabhupadaDictionary.qrc">
    <normaloff>:/Icon/resources/PrabhupadaDictionary.ico</normaloff>:/Icon/resources/PrabhupadaDictionary.ico</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="LayoutPrabhupadaLogin">
   <item>
    <layout class="QHBoxLayout" name="LayoutSQL">
     <item>
      <widget class="QRadioButton" name="radioButtonSQLite">
       <property name="text">
        <string>SQLite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonPostgreSQL">
       <property name="text">
        <string>PostgreSQL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="LayoutConnection">
     <item row="0" column="0">
      <widget class="QLabel" name="LabelUserName">
       <property name="text">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="ComboBoxUserName">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelPassword">
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LineEditPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelDatabaseName">
       <property name="text">
        <string>Имя базы данных</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ComboBoxDatabaseName">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelHostName">
       <property name="text">
        <string>Сервер</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="ComboBoxHostName">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelPort">
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="ComboBoxPort">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="LabelSchema">
       <property name="text">
        <string>Схема</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="ComboBoxSchema">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="CheckBoxResetSettings">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Этот флаг сбрасывает все настройки пользователя после входа в программу! Но только те из них, которые хранятся в базе данных! Настройки, хранящиеся в файлах &amp;quot;ini&amp;quot; сбрасывайте простым удалением этих файлов!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Сбросить настройки</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="LabelLanguageUI">
       <property name="text">
        <string>Язык программы</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="ComboBoxLanguageUI"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>Поехали!</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Задний ход!</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonSQLite</tabstop>
  <tabstop>radioButtonPostgreSQL</tabstop>
  <tabstop>ComboBoxUserName</tabstop>
  <tabstop>LineEditPassword</tabstop>
  <tabstop>ComboBoxDatabaseName</tabstop>
  <tabstop>ComboBoxHostName</tabstop>
  <tabstop>ComboBoxPort</tabstop>
  <tabstop>ComboBoxSchema</tabstop>
  <tabstop>CheckBoxResetSettings</tabstop>
 </tabstops>
 <resources>
  <include location="PrabhupadaDictionary.qrc"/>
 </resources>
 <connections/>
</ui>
